x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/customization_modal.tsx

```diff
@@ -0,0 +1,159 @@
+
+  return (
+    <EuiOverlayMask>
+      <EuiModal onClose={onClose}>
+        <EuiModalHeader>
+          <EuiModalHeaderTitle>
+            {i18n.translate(
+              'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.title',
+              {
+                defaultMessage: 'Customize document search',
+              }
+            )}
+          </EuiModalHeaderTitle>
+        </EuiModalHeader>
+        <EuiModalBody>
+          <EuiForm>
+            <EuiFormRow
+              label={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.filterFieldsLabel',
+                {
+                  defaultMessage: 'Filter fields',
+                }
+              )}
+              fullWidth={true}
+              helpText={
+                <EuiText>
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.filterFields',
+                    {
+                      defaultMessage:
+                        'Faceted values rendered as filters and available as query refinement',
+                    }
+                  )}
+                </EuiText>
+              }
+            >
+              <EuiComboBox
+                data-test-subj="filterFieldsDropdown"
+                fullWidth={true}
+                options={selectableFilterFields}
+                selectedOptions={selectedFilterFields}
+                onChange={setSelectedFilterFields}
+              />
+            </EuiFormRow>
+            <EuiFormRow
+              label={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.sortFieldsLabel',
+                {
+                  defaultMessage: 'Sort fields',
+                }
+              )}
+              fullWidth={true}
+              helpText={
+                <EuiText>
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.sortFields',
+                    {
+                      defaultMessage:
+                        'Used to display result sorting options, ascending and descending',
+                    }
+                  )}
+                </EuiText>
+              }
+            >
+              <EuiComboBox
+                data-test-subj="sortFieldsDropdown"
+                fullWidth={true}
+                options={selectableSortFields}
+                selectedOptions={selectedSortFields}
+                onChange={setSelectedSortFields}
+              />
+            </EuiFormRow>
+          </EuiForm>
+        </EuiModalBody>
+        <EuiModalFooter>
+          <EuiButtonEmpty onClick={onClose}>
+            {i18n.translate(
+              'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.cancel',
+              {
+                defaultMessage: 'Cancel',
```

Comment:
[not a change request, just me thinking out loud] I really gotta DRY these simple "Cancel"/"Edit"/"Delete" etc type strings out into a shared constants file 🤔